curriculum/challenges/english/01-responsive-web-design/learn-css-transforms-by-building-a-penguin/step-002.md

```diff
@@ -0,0 +1,56 @@
+---
+id: 61967e74a8e3690ab6292daa
+title: Step 2
+challengeType: 0
+dashedName: step-2
+---
+
+# --description--
+
+Target the `body` element to set the `background` to a linear gradient angled 45 degrees clockwise, starting at `rgb(118, 201, 255)` and ending at `rgb(247, 255, 222)`.
+
+# --hints--
+
+You should use the `body` element selector.
+
+```js
+assert.match(code, /body\s*\{/);
+```
+
+You should use the `background` property in the `body` selector.
+
+```js
+assert.isTrue(new __helpers.CSSHelp(document).isPropertyUsed('background'));
+```
+
+You should set `background` to `linear-gradient(45deg, rgb(118, 201, 255), rgb(247, 255, 222))`.
+
+```js
+assert.equal(new __helpers.CSSHelp(document).getStyle('body')?.getPropVal('background', true), 'linear-gradient(45deg,rgb(118,201,255),rgb(247,255,222))');
```

Comment:
```suggestion
assert.include(['linear-gradient(45deg,rgb(118,201,255),rgb(247,255,222))', 'rgba(0,0,0,0)linear-gradient(45deg,rgb(118,201,255),rgb(247,255,222))repeatscroll0%0%'], new __helpers.CSSHelp(document).getStyle('body')?.getPropVal('background', true));
```

Reply:
Crazy that it differs this much, though....

Reply:
After digging on it yesterday, I believe the reason is that firefix puts all? or most? of the background related properties into `background`, whether you set them or not. background-color, -image, -repeat, etc.